docs/v3-ea-components/transport-types/http-transport.md

```diff
@@ -1,3 +1,135 @@
 # HTTP Transport
 
-_Guide coming soon_
+HttpTransport is used to fetch data from a Provider using HTTP(S) protocol. In order to use HttpTransport you need to provide two methods `prepareRequests` and `parseResponse` that will be used to build requests and parse responses respectively. 
+
+Example usage of HttpTransport
+```typescript  
+const transport = new HttpTransport<EndpointTypes>({  
+  prepareRequests: (params, config) => {  
+    return params.map((param) => {  
+      const symbol = param.symbol.toLowerCase()  
+      const url = `/price/${symbol}`  
+  
+      return {  
+        params: param  
+		request: {  
+          baseURL: config.API_ENDPOINT,  
+          url,  
+        },  
+      }  
+    })  
+  },  
+  parseResponse: (params, res, config) => {  
+    return res.data.map((result) => {  
+      return {  
+        params: { symbol: result.symbol},  
+        response: {  
+          data: {  
+            result: result.price,  
+          },  
+          result: result.price,  
+        },  
+      }  
+    })  
+  },  
+})
+```
+### Building requests
+**prepareRequests**  method  takes the list of currently active input parameters in the subscription set and adapter config and should return one or multiple request configs (_ProviderRequestConfig_) depending if Data Provider supports request batching or not.
+
+#### Request config structure
+*ProviderRequestConfig* is a object that contains two keys `params` and `request` . `params` is the input parameters for a request and whatever  is provided will be available in the **parseResponse** method. `request` is the *axios* request configuration where request url, request method, query or body params, headers and etc can be specified.
```

Comment:
> `params` is the EA input parameters for a request

This^ is accurate, right?

```suggestion
*ProviderRequestConfig* is an object with two keys: `params` and `request`. `params` is the EA input parameters for a request, and will also be available in the **parseResponse** method. `request` is the *axios* request configuration where parameters such as the request URL, request method, query or body params, and headers can be specified.
```

Reply:
regarding `params` i would say yes and. no :). it will be more accurate to say
 `params` is the non-expired input parameters in the subscription set, but i think it will be more confusing or somewhere we need to also mention what is that subscription set and how it works.

Reply:
Ok, I'm fine with how it is now!